tests/letstest/scripts/test_letsencrypt_auto_certonly_standalone.sh

```diff
@@ -7,10 +7,32 @@
 #public_ip=$(curl -s http://169.254.169.254/2014-11-05/meta-data/public-ipv4)
 #private_ip=$(curl -s http://169.254.169.254/2014-11-05/meta-data/local-ipv4)
 
-cd letsencrypt/letsencrypt-auto-source
-./letsencrypt-auto --os-packages-only --debug --version
-./letsencrypt-auto certonly --no-self-upgrade -v --standalone --debug \
+cd letsencrypt
+export PATH="$PWD/letsencrypt-auto-source:$PATH"
+letsencrypt-auto --os-packages-only --debug --version
+letsencrypt-auto certonly --no-self-upgrade -v --standalone --debug \
                    --text --agree-dev-preview --agree-tos \
                    --renew-by-default --redirect \
                    --register-unsafely-without-email \
                    --domain $PUBLIC_HOSTNAME --server $BOULDER_URL
+
+# we have to jump through some hoops to cope with relative paths in renewal
+# conf files ...
+# 1. be in the right directory
+cd tests/letstest/testdata/
+
+# 2. refer to the config with the same level of relativitity that it itself
+# contains :/
+letsencrypt-auto certificates --config-dir sample-config
```

Comment:
I tried doing exactly that, but it didn't work -- shell variables aren't greppable multi-line objects.  This line is there so that someone running the test farm tests can see what the output of the `certbot certificates` command was.